llvm_util/llvm2alive.cpp

```diff
@@ -711,14 +712,47 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
           }
         } else if (name == "align") {
           llvm::Value *ptr = bundle.Inputs[0].get();
-          llvm::Value *align = bundle.Inputs[1].get();
-          if (bundle.Inputs.size() != 2)
-            // TODO: "align" assume operand bundle may take a third argument.
+          auto *aptr = get_operand(ptr);
+          if (!aptr)
             return error(i);
 
-          auto *aptr = get_operand(ptr), *aalign = get_operand(align);
-          if (!aptr || !aalign)
+          auto *align = llvm::cast<llvm::ConstantInt>(bundle.Inputs[1].get());
+          if (align->uge(UINT64_MAX))
             return error(i);
+          uint64_t align_value = align->getZExtValue();
+          auto *aalign = make_intconst(align_value, 64);
```

Comment:
this change doesn't make sense. the code above is equivalent to get_operand. Without caching.

Reply:
Then should we change `Assume::toSMT` instead? `AlignmentFromAssumptions/simple.ll` is crashing due to 128 bit alignment :/

Reply:
The difference is its bit size: `getInt()` crashes with larger-than-64bits int

Reply:
I see. These 128-bit tests are artificial; we can leave them as unsupported.
I would just revert part of the change above so it continues to use `get_operand`. And leave a check to bail out if bits > 64.